go/test/endtoend/vtgate/foreignkey/unsharded_vschema.json

```diff
@@ -19,6 +19,23 @@
     "fk_t16": {},
     "fk_t17": {},
     "fk_t18": {},
-    "fk_t19": {}
+    "fk_t19": {},
+    "fk_t20": {},
+    "fk_multicol_t1": {},
+    "fk_multicol_t2": {},
+    "fk_multicol_t3": {},
+    "fk_multicol_t4": {},
+    "fk_multicol_t5": {},
+    "fk_multicol_t6": {},
+    "fk_multicol_t7": {},
+    "fk_multicol_t10": {},
+    "fk_multicol_t11": {},
+    "fk_multicol_t12": {},
+    "fk_multicol_t13": {},
+    "fk_multicol_t15": {},
+    "fk_multicol_t16": {},
+    "fk_multicol_t17": {},
+    "fk_multicol_t18": {},
+    "fk_multicol_t19": {}
   }
 }
```

Comment:
nit: how about not providing vschema for unsharded and letting the schema tracking add those tables with fk information?

Reply:
Just the table names are there, we anyways wait for fk information to be present.